roottest/root/meta/cmsUnload/CMakeLists.txt

```diff
@@ -1,8 +1,43 @@
-#-------------------------------------------------------------------------------
-#
-# Placeholder file to translate the tests to the new CTest system. Meanwhile we
-# define a CTest test that calls 'make' in ${CMAKE_CURRENT_SOURCE_DIR}
-#
-#-------------------------------------------------------------------------------
-
-ROOTTEST_ADD_OLDTEST()
+include_directories(${CMAKE_CURRENT_SOURCE_DIR}/inc)
+
+ROOTTEST_GENERATE_REFLEX_DICTIONARY(Edm
+                                    inc/Wrapper.h
+                                    SELECTION inc/edm_selection.xml
+                                    FIXTURES_SETUP root-meta-cmsUnload-Edm-fixture)
+
+ROOTTEST_GENERATE_REFLEX_DICTIONARY(Strip
+                                    inc/SiStripCluster_classes.h
+                                    SELECTION inc/strip_selection.xml
+                                    FIXTURES_SETUP root-meta-cmsUnload-Strip-fixture)
+
+ROOTTEST_GENERATE_REFLEX_DICTIONARY(Cluster
+                                    inc/ClusterTotals.h
+                                    SELECTION inc/cluster_selection.xml
+                                    FIXTURES_SETUP root-meta-cmsUnload-Cluster-fixture)
+
+
+macro(CMS_UNLOAD_TEST name use_rmap)
+  if(${use_rmap} STREQUAL "true")
+    set(testname rmap-${name})
+  else()
+    set(testname ${name})
+  endif()
+
+  ROOTTEST_ADD_TEST(${testname}
+                    MACRO runscript.C
+                    COPY_TO_BUILDDIR step2.cxx
```

Comment:
Since all the tests are copying the files, is there a risk to run into a race condition (i.e. a copy going at the same time a use, leading to (very rare) failure due to incomplete file)?